actions/config_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * config_test.go - tests for creating the config file
+ *
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package actions
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+// Test that the global config file is created with mode 0644, regardless of the
+// current umask.
+func TestConfigFileIsCreatedWithCorrectMode(t *testing.T) {
+	oldMask := unix.Umask(0)
+	defer unix.Umask(oldMask)
+	unix.Umask(0077)
+
+	tempFile, err := ioutil.TempFile("", "fscrypt_conf")
+	if err != nil {
+		t.Fatal(err)
+	}
+	ConfigFileLocation = tempFile.Name()
+	os.Remove(ConfigFileLocation)
```

Comment:
Do we need this `os.Remove` and the `defer os.Remove` below?

It seems like it might be better to just use `ioutil.TempDir` and then cleanup everything in a single call.

Reply:
Good idea.  I changed it to do that instead.